src/redux/data.ts

```diff
@@ -1191,8 +1194,8 @@ export const addressAssetsReceived = (
   const assetsWithScamURL: string[] = Object.values(parsedAssets)
     .filter(
       asset =>
-        (isValidDomain(asset.name.replaceAll(' ', '')) ||
-          isValidDomain(asset.symbol)) &&
+        ((asset?.name && isValidDomain(asset?.name.replaceAll(' ', ''))) ||
+          (asset?.symbol && isValidDomain(asset.symbol))) &&
```

Comment:
fixed https://sentry.io/organizations/rainbow-me/issues/3445380812/?project=1855565&query=release%3Ame.rainbow%401.6.17%2B31&statsPeriod=14d

Reply:
and https://sentry.io/organizations/rainbow-me/issues/3442735980/?project=1855565&query=release%3Ame.rainbow%401.6.17%2B31&statsPeriod=14d